.coveragerc

```diff
@@ -27,6 +27,7 @@ omit =
     homeassistant/components/adguard/sensor.py
     homeassistant/components/adguard/switch.py
     homeassistant/components/ads/*
+    homeassistant/components/advantage_air/diagnostics.py
```

Comment:
Seems a shame to exclude tests

Reply:
I cant find any documentation on this feature, so I just copied https://github.com/home-assistant/core/pull/64576 and https://github.com/home-assistant/core/pull/64374, neither of which have tests.

Reply:
There is a test helper (`get_diagnostics_for_config_entry`) in `tests.components.diagnostics` you can use